code/src/WijmoProvider/Columns/Factory.ts

```diff
@@ -67,7 +67,12 @@ namespace WijmoProvider.Column {
                         editorConfigs
                     );
                 case OSFramework.Enum.ColumnType.Text:
-                    return new TextColumn(grid, columnID, configs);
+                    return new TextColumn(
+                        grid,
+                        columnID,
+                        configs,
+                        editorConfigs
```

Comment:
what do you think about changing the name of the parameter editorConfigs to extraConfigs?

Reply:
agreed